ChatSecure/Classes/OTRAppDelegate.m

```diff
@@ -569,7 +569,9 @@ - (void) userNotificationCenter:(UNUserNotificationCenter *)center willPresentNo
 
 - (void) userNotificationCenter:(UNUserNotificationCenter *)center didReceiveNotificationResponse:(UNNotificationResponse *)response withCompletionHandler:(void (^)())completionHandler {
     NSDictionary *userInfo = response.notification.request.content.userInfo;
-    if (userInfo[kOTRNotificationThreadKey] == nil) {
+    if ([kOTRNotificationTypeNone isEqualToString:userInfo[kOTRNotificationType]]) {
+        // Nothing
+    } else if ([kOTRNotificationTypeSubscriptionRequest isEqualToString:userInfo[kOTRNotificationType]]) {
```

Comment:
From the Apple docs for `NSString.h`:

```objc
NS_ASSUME_NONNULL_BEGIN
...
- (BOOL)isEqualToString:(NSString *)aString;
```

The method assumes `aString` is nonnull, so even if it doesn't crash, it's not what the API wants. For instance, in Swift it wouldn't even compile. It would be better to at least swap the order so you're sending a message to nil, instead of passing a nil parameter.